### PR TITLE
Escape cdata_regex

### PR DESCRIPTION
**Pull request by honeybadgerdontcare**
`\` needs to be escaped for regex, updated cdata_regex to `\\w+`.

### CODE REVIEW

**Comment by honeybadgerdontcare** [comment]
@aghassemi added a test file that showed it needed some more tweaks to the regex. PTAL.

File: extensions/amp-timeago/0.1/test/validator-amp-timeago.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <script async custom-element="amp-timeago" src="https://cdn.ampproject.org/v0/amp-timeago-latest.js"></script>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+
+    <!-- valid -->
+    <amp-timeago layout="fixed" width="160" height="20" datetime="2017-04-11T00:37:33.809Z" locale="en">Saturday 11 April 2017 00.37</amp-timeago>
+
+    <!-- valid, locale optional -->
+    <amp-timeago layout="fixed" width="160" height="20" datetime="2017-04-11T00:37:33.809Z">Saturday 11 April 2017 00.37</amp-timeago>
```
Comment (aghassemi) [inline extensions/amp-timeago/0.1/test/validator-amp-timeago.html]: you want this without `Z` right?

Comment (aghassemi) [inline extensions/amp-timeago/0.1/test/validator-amp-timeago.html]: never mind

**Comment by aghassemi** [comment]
LGTM

**Pull request merged**